web/static/scss/_components.scss

```diff
@@ -115,6 +115,10 @@
   a.btn {
     margin: 0.7em 0 3em 0;
   }
+
+  form {
```

Comment:
Maybe we should make it a specific name because it's something we don't really control, so like .phoenix-button-form?